glass/grf/_transformations.py

```diff
@@ -0,0 +1,178 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+
+# type imports for NDArray[Any] cannot be moved into TYPE_CHECKING here
+# otherwise, the dispatch mechanism cannot resolve the class dynamically
+from typing import Any
+
+from numpy.typing import NDArray  # noqa: TC002
```

Comment:
Why not do as elsewhere and not have the `from`?

Reply:
See above, I rather have the types available as plain symbols